server/app.js

```diff
@@ -0,0 +1,48 @@
+const createError = require("http-errors");
+const express = require("express");
+const path = require("path");
+const cookieParser = require("cookie-parser");
+const logger = require("morgan");
+const db = require("./database/models/index.js");
+
+const indexRouter = require("./routes/index");
+const usersRouter = require("./routes/users");
+const techlistRouter = require("./routes/techlist");
+const submitvoteRouter = require("./routes/submitvote");
+
+const app = express();
+
+//connect db
+db.connector.sync();
+
+app.set("views", path.join(__dirname, "views"));
+app.set("view engine", "jade");
```

Comment:
Not sure if the use of Jade is on purpose, but in case you were not entirely sure about Jade, there are also a lot of other template engines. Personally I used to like EJS and Handlebars :)